fonts.css

```diff
@@ -21,4 +21,11 @@
     src: url('fonts/ptserif-regular.woff') format('woff');
     font-weight: normal;
     font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto';
+    src: url('fonts/Roboto-Regular.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)